tokio/src/time/error.rs

```diff
@@ -64,7 +64,9 @@ impl fmt::Display for Error {
     fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
         use self::Kind::*;
         let descr = match self.0 {
-            Shutdown => "timer is shutdown",
+            Shutdown => {
+                "the timer is shutdown, this code must be executed from within a tokio runtime"
```

Comment:
This message is great, I think though this is actually the place where we want to add the panic message as well https://github.com/tokio-rs/tokio/blob/master/tokio/src/time/driver/handle.rs#L24

I am also curious if this is something we could test? maybe via a panic handler?

Reply:
Hey @LucioFranco - I've added the other panic, and also update the text to match another similar panic code I found elsewhere in the code base. There are now 4 panic messages that include `must be called from the context of Tokio runtime`

I also added one test case for when there is no timer, but I'm having a hard time reproducing the other panic messages. I think the tests should look very similar to the one I have written, I just can't seem to speak the magic incantation to get it to actually panic that way.